bedrock/firefox/templates/firefox/family/includes/download-firefox.html

```diff
@@ -6,10 +6,9 @@
 
 <section class="c-download-firefox mzp-t-dark">
   <div class="mzp-l-content mzp-t-content-md">
-    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-firefox">Firefox</div>
+    <div class="mzp-c-wordmark mzp-t-wordmark-lg mzp-t-product-firefox">Firefox</div>
 
-    <!-- TODO: replace heart with inline image and alt? Maybe a span with image replacement? An actual emoji character? -->
-    <h2>We (heart emoji) all families</h2>
+    <h2 class="c-download-firefox-title">We <img src="{{ static('img/firefox/family/icon-heart-outline.svg') }}" alt="heart"> all families</h2>
```

Comment:
**nit**: The heart image rendered here seems about 1.5 the size from what is in figma